mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala

```diff
@@ -256,4 +256,38 @@ trait HasFitIntercept extends Params {
   /** @group getParam */
   final def getFitIntercept: Boolean = getOrDefault(fitIntercept)
 }
+
+/**
+ * :: DeveloperApi ::
+ * Trait for shared param elasticNetParam.
+ */
+@DeveloperApi
+trait HasElasticNetParam extends Params {
```

Comment:
do you suggest to move it to LinearRegression.scala? we will use it in LOR as well.